internal/cmd/config/config.go

```diff
@@ -337,12 +339,58 @@ func Parse(d string) (*Config, error) {
 		if !strutil.Printable(result.Worker.Name) {
 			return nil, errors.New("Worker name contains non-printable characters")
 		}
+
 		if result.Worker.TagsRaw != nil {
 			switch t := result.Worker.TagsRaw.(type) {
+			// We allow `tags` to be a simple string containing a reference to
+			// an environment variable (env://) / file path (file://).
```

Comment:
parseutil currently supports `env://` and `file://`, but it is possible that will be expanded, at which point this comment would get stale, perhaps rephrase this to something like:

```go
// We allow `tags` to be a simple string containing a URL with schema.
// See: https://github.com/hashicorp/go-secure-stdlib/blob/main/parseutil/parsepath.go
```

Reply:
Good point, thank you!

Reply:
Addressed in 9e383a9